_studio/shared/src/libmfx_allocator_vaapi.cpp

```diff
@@ -42,6 +42,11 @@
 #define VASurfaceAttribUsageHint 8
 #define VAEncMacroblockMapBufferType 29
 
+// TODO: remove this internal definition once it appears in VAAPI
+#define VA_FOURCC(ch0, ch1, ch2, ch3) \
```

Comment:
MFX_MAKEFOURCC should be used. Isn't it the same?
What is the reason to introduce HERE local VA_FOURCC_R5G6B5? Isn't it defined elsewhere?

Reply:
VA_FOURCC_R5G6B5 is not defined in VA API.